### DISCUSSION TITLE
Need some help. Replacement for failed intellitouch

### DISCUSSION THREAD

**Discussion by triorbonk** [tagyoureit/nodejs-poolController#681] [Nixie] [answered]
I have used NPC for a few years now and love it.   My intellitouch is now having trouble witrh the rs485 network. I would like to have my NPC control the pumps directly but when I unplug the intellitouch there is no traffic on the rs485 network. I have reinstalled the NCP with no success. The client never let's me set the mode to nixie 2 body.

**Comment by triorbonk** [comment, marked as answer]
I was able to get Nixie working.  Just needed to wait a bit for the controller to initialize and start talking to the client.  Thanks, guys!

**Reply by rstrouse** [reply]
Yep she just needs to get her footing the first time then it will initialize the pumps more quickly from then on.

**Reply by triorbonk** [reply]
Thanks for the quick response.  My pool is now limping along.  I just ordered a handful of parts to extend the nixie controller and replace the intelllitouch.  

One question, can I use the existing 24VAC power supply in the Pentair cabinet?

**Reply by rstrouse** [reply]
I believe you can.  You should tap it after the low voltage breakers for the 24v supply for overload protection.  I don't run an IntelliTouch anymore and I do not remember the breaker layout but I think one of them is specifically for the valve output.  You should also be able to put a rectifier on that output for 24vdc for your relay control.

**Reply by triorbonk** [reply]
I will try this and report back.  Thanks
